.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-  - '8'
   - '10'
+  - '12'
```

Comment:
what about using `node` or `lts/*` per https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions if we just want to keep the node version up to date?

not sure what the use of listing two versions are either wouldn't it just install one or the other always

Reply:
no test script? cant comment on line 16 weirdly but thats what this comment is referring to

Reply:
the build script runs the test suite, also yeah `lts/*` is a better idea

Reply:
ahh i see, didn't know about the prebuild. for sure.